packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -1828,6 +1829,107 @@ void main() {
     expect(RendererBinding.instance!.mouseTracker.debugDeviceActiveCursor(1), SystemMouseCursors.click);
   });
 
+  group('feedback', () {
+    late FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback.dispose();
+    });
+
+    Widget feedbackBoilerplate({bool? enableFeedback}) {
+      return  MaterialApp(
+        home: Scaffold(
+          bottomNavigationBar: BottomNavigationBar(
+            enableFeedback: enableFeedback,
+            items: const <BottomNavigationBarItem>[
+              BottomNavigationBarItem(icon: Icon(Icons.ac_unit), title: Text('AC')),
+              BottomNavigationBarItem(icon: Icon(Icons.access_alarm), title: Text('Alarm')),
+            ],
+          ),
+        ),
+      );
+    }
```

Comment:
Is this one necessary? The one with the theme below is just using null for `enableFeedbackTheme` unless it is set, which the default theme above would have. I don't think there needs to be two of these.